db/re/item_db.txt

```diff
@@ -11534,14 +11597,139 @@
 23196,Agust_Lucky_Scroll,Shining Blue Lucky Egg,18,10,,10,,,,0,0xFFFFFFFF,63,2,,,1,,,{ getgroupitem(IG_Agust_Lucky_Scroll); },{},{}
 //
 23228,Hazy_Mooncake,Hazy Mooncake,0,768,,10,,,,,0xFFFFFFFF,63,2,,,,,,{ /* unknown */ },{},{}
-//
+23229,Scroll_of_Fallen_Angel_Wings,Scroll of Fallen Angel Wings,11,,,10,,,,,0xFFFFFFFF,63,2,,,,,,{ upgradeui 23229; },{},{}
+23236,Class_Shadow_Box_Weapon,Class Shadow Box (Weapon),11,,,10,,,,,0xFFFFFFFF,63,2,,,,,,{ synthesisui 23236; },{},{}
+23237,Class_Shadow_Box_Armor,Class Shadow Box (Armor),11,,,10,,,,,0xFFFFFFFF,63,2,,,,,,{ synthesisui 23237; },{},{}
+23238,Class_Shadow_Box_Shoes,Class Shadow Box (Shoes),11,,,10,,,,,0xFFFFFFFF,63,2,,,,,,{ synthesisui 23238; },{},{}
+23239,Class_Shadow_Box_Shield,Class Shadow Box (Shield),11,,,10,,,,,0xFFFFFFFF,63,2,,,,,,{ synthesisui 23239; },{},{}
+23240,Class_Shadow_Box_Pendant,Class Shadow Box (Pendant),11,,,10,,,,,0xFFFFFFFF,63,2,,,,,,{ synthesisui 23240; },{},{}
+23241,Class_Shadow_Box_Earring,Class Shadow Box (Earring),11,,,10,,,,,0xFFFFFFFF,63,2,,,,,,{ synthesisui 23241; },{},{}
+23247,StatusShadow_Mix,StatusShadow Mix,11,,,10,,,,,0xFFFFFFFF,63,2,,,,,,{ synthesisui 23247; },{},{}
+23248,GemstoneShadow_Mix,GemstoneShadow Mix,11,,,10,,,,,0xFFFFFFFF,63,2,,,,,,{ synthesisui 23248; },{},{}
+23249,BearersShadow_Mix,BearerShadow Mix,11,,,10,,,,,0xFFFFFFFF,63,2,,,,,,{ synthesisui 23249; },{},{}
+23250,ComposeShadow_Mix,ComposerShadow Mix,11,,,10,,,,,0xFFFFFFFF,63,2,,,,,,{ synthesisui 23250; },{},{}
 23256,Elixir_Bandages,Elixir Bandages,2,10,,0,,,,,0xFFFFFFFF,63,2,,,,,,{ pet 1041; },{},{}
 23277,Mado_Box,Emergency Magic Gear,2,10000,,3000,,,,,0x00000400,56,2,,,100,,,{ setmadogear 1; },{},{}
 23280,N_Fly_Wing_,Novice Fly Wing,11,10,,0,,,,,0xFFFFFFFF,63,2,,,1:98,,,{ itemskill "AL_TELEPORT",1; },{},{}
+23281,RaceShadow_Mix,RaceShadow Mix,11,,,10,,,,,0xFFFFFFFF,63,2,,,,,,{ synthesisui 23281; },{},{}
 23288,Compressed_Wing_Of_Fly,Compressed Fly Wing,11,1000,,10,,,,,0xFFFFFFFF,63,2,,,,,,{ itemskill "AL_TELEPORT",1; },{},{}
+23289,Candy_Pouch_Blessing_Scroll_Melee,Candy Pouch Blessing Scroll(Physical),11,,,10,,,,,0xFFFFFFFF,63,2,,,,,,{ upgradeui 23289; },{},{}
```

Comment:
should be syntehsisui as its not on the upgradebox.lub

also applies to 23290 and 91

Reply:
thanks
some boxes were written in upgrade and synthesis. seems there are patches after my initial work, as you can see on my notes item_synthesis.yml. I was wondering why this item have note "the cards & refine will be removed" :rofl:

Reply:
23229 also has the same problem. hehe.

Reply:
the data,grf & rdata.grf have difference, that cause this duplication, so I will this as is. user can switch which one will be used (since those have similiar result) until kRO clean it themself